src/components/10-atoms/text/react/DemoTextReact.jsx

```diff
@@ -9,19 +9,25 @@ const DemoTextReact = () => {
     setText(!text);
   };
 
-  const handleCounter = ev => {
+  const handleCounter = () => {
     setCounter(counter + 1);
   };
 
   return (
     <div>
-      <label className="js-pure-text">
+      <label htmlFor="checkbox" className="js-pure-text">
```

Comment:
wow, I had no idea this was needed for accessible React, so cumbersome - glad you caught it here and elsewhere!